posthog/api/event.py

```diff
@@ -212,6 +213,9 @@ def get_values(self, request: request.Request) -> List[Dict[str, Any]]:
             where = ""
 
         params.append(self.team_id)
+        params.append(relative_date_parse("-7d").strftime("%Y-%m-%d 00:00:00"))
+        params.append(timezone.now().strftime("%Y-%m-%d 00:00:00"))
```

Comment:
if a user joins today it means they'll get 0 results even if they have sent events?

Reply:
yeah you'r right the time shoudl extend to the entire current day